src/Core/Enums/CollectionType.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Bit.Core.Enums;
+
+public enum CollectionType
+{
+    Shared,
+    DefaultUserCollection,
```

Comment:
We tend to expressly number our enums, which I also prefer.

For consistency, I also suggest:
- `SharedCollection` rather than just `Shared`
- either `DefaultUserCollection` or `UserDefaultCollection` but not both - it is currently inconsistent between this and the db column. I think I was inconsistent when writing the tickets. `DefaultUserCollection` reads a little better to me but I don't feel strongly about it.